main/SQLHelper.cpp

```diff
@@ -2428,6 +2428,10 @@ bool CSQLHelper::OpenDatabase()
 			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviSmartdenIPInOut);
 			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviSmartdenLan);
 		}
+		if (dbversion < 130)
```

Comment:
Something seems not right here
you placed a check for version 130 above 124 ?
Could you place it where it belongs ?
You also need to increase the actual database version with 1 at the top of the class